package.json

```diff
@@ -112,5 +54,64 @@
     "when": "^3.7.4",
     "xml2js": "^0.4.17",
     "yargs": "^3.27.0"
+  },
+  "devDependencies": {
+    "babel": "^6.5.2",
+    "babel-core": "^5.8.38",
+    "babel-eslint": "^7.2.2",
+    "babelify": "^6.4.0",
+    "browser-sync": "^2.7.13",
+    "browserify": "^10.2.6",
+    "debowerify": "^1.3.1",
+    "del": "^1.2.0",
+    "eslint": "^3.19.0",
+    "eslint-config-airbnb": "^14.1.0",
+    "eslint-plugin-import": "^2.2.0",
+    "eslint-plugin-jsx-a11y": "^4.0.0",
+    "eslint-plugin-react": "^6.10.3",
+    "gulp": "^3.9.0",
+    "gulp-autoprefixer": "^2.3.1",
+    "gulp-if": "^1.2.5",
+    "gulp-imagemin": "^2.3.0",
+    "gulp-jsx-coverage": "^0.2.6",
+    "gulp-mocha": "^2.1.3",
+    "gulp-notify": "^2.2.0",
+    "gulp-rename": "^1.2.2",
+    "gulp-sourcemaps": "^1.5.2",
+    "gulp-streamify": "0.0.5",
+    "gulp-uglify": "^1.2.0",
+    "gulp-util": "^3.0.6",
+    "history": "^3.2.1",
+    "humps": "^0.6.0",
+    "istanbul": "^0.4.0",
+    "jsdom": "^11.6.2",
+    "meta-tag-data": "^1.1.0",
+    "mocha": "^2.3.3",
+    "postcss": "^5.0.16",
+    "postcss-modules-extract-imports": "^1.0.0",
+    "postcss-modules-local-by-default": "^1.0.1",
+    "postcss-modules-scope": "^1.0.0",
+    "postcss-modules-values": "^1.1.1"
+  },
+  "keywords": [
+    "boilerplate",
+    "browserify",
+    "gulp",
+    "react"
+  ],
+  "private": true,
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/blockstack/blockstack.org.git"
+  },
+  "scripts": {
+    "build-docs": "npm run fetch-docs; gulp buildDocs",
+    "dev": "npm run fetch-docs; gulp dev",
+    "start": "serve ./build --single",
+    "fetch-docs": "git submodule init; git submodule update --remote",
+    "postinstall": "cd ./node_modules/highlight.js && npm i && node ./tools/build.js :common",
+    "build": "npm run fetch-docs; gulp prod",
+    "prod": "npm run fetch-docs; gulp prod",
+    "format": "prettier --write \"app/**/*.js\" && echo -e '\\033[0;32m'💅 Just use prettier'\\033[0m'"
   }
```

Comment:
Why'd you reorganize the package.json file (or was it prettier?)? The standard is scripts near the top before dependencies. Can we change that back? If we wanna reorganize we can always make another PR to discuss.

Reply:
For every new project I work on I run this on it: https://github.com/henrikjoreteg/fixpack

It organizes them in a consistent and predictable way, and if I ever modify a `package.json `manually, I like to run it to keep things tidy.

```
It will re-write your package.json file as follows:

name first
description second
version third
author fourth
all other keys in alphabetical order
dependencies and devDependencies sorted alphabetically
newline at the end of the file

It will warn you if any of these are missing:

description
author
repository
keywords
main
bugs
homepage
license
```

I don't know that this really warrants a new, separate PR, if there is a rational or previously agreed upon organizational structure of `package.json` files for blockstack, let me know and I'll switch it back!